src/main/java/com/microsoft/sqlserver/jdbc/Geometry.java

```diff
@@ -104,8 +121,8 @@ public static Geometry deserialize(byte[] wkb) throws SQLServerException {
     }
 
     /**
-     * Returns a Geometry instance from an Open Geospatial Consortium (OGC) Well-Known Text (WKT) representation. SRID
-     * is defaulted to 0.
+     * Constructor for a Geometry instance from an Open Geospatial Consortium (OGC) Well-Known Text (WKT)
+     * representation. Spatial Reference Identifier is defaulted to 0.
      * 
      * @param wkt
```

Comment:
This method has param name as `wkt`, the other method has `wellKnownText`, can we generalize and use same name for all similar parameters?

Reply:
done